ocaml/libs/http-lib/dune

```diff
@@ -45,6 +45,7 @@
     ipaddr
     polly
     threads.posix
+    tracing
```

Comment:
All packages using this library need to have their opam dependencies checked: they will need to add `xapi-tracing` to them:

```
ocaml/database/dune
36:    httpsvr
80:    httpsvr
```
This is used only by binaries in the package xapi

```
ocaml/libs/http-lib/dune
38:  (name httpsvr)
127:    httpsvr
```
same, for testing the xapi package.

```
ocaml/xcp-rrdd/bin/rrdd/dune
12:    httpsvr
51:    httpsvr
```
This are in the xapi-rrdd package.

```
ocaml/tests/dune
21:    httpsvr
66:    httpsvr
```
These are tests in the xapi package.

```
ocaml/xapi-cli-server/dune
8:    httpsvr
```
This library is only used in the xapi package.

```
ocaml/xapi/dune
64:    httpsvr
118:    httpsvr
231:    httpsvr
```
These are all part of the xapi package.

So two packages are affected by this change, xapi and xapi-rrdd. The first already declares tracing as a dependency, but xapi-rrdd doesn't. It's definition needs to be change to depend on xapi-tracing, this is in dune-project. Remember to run make as well: the corresponding xapi-rrdd will be modified accordingly by dune.

Reply:
This https://github.com/xapi-project/xen-api/pull/5888/commits/9e5acaa43f55e99e25210e7002b0a462024ea244?

Reply:
yeah, that works